wrappers/desktop.go

```diff
@@ -61,6 +61,13 @@ var validDesktopFilePrefixes = []string{
 	"Keywords=",
 	"StartupNotify=",
 	"StartupWMClass=",
+	// unity extension
+	"X-Ayatana-Desktop-Shortcuts=",
+	"TargetEnvironment=",
+}
+
+var validDesktopFileSuffixes = []string{
+	"Shortcut Group]",
```

Comment:
I was initially confused by the unpaired `]`, maybe add a simple comment here:
`// NOTE: The absence of leading square brace is intentional`.